README.md

```diff
@@ -110,7 +110,7 @@ pravega_client_auth_loadDynamic=true
 
 -  You must make the Maven repo in SDP available to your development workstation.
 ```
-kubectl port-forward service/repo 9090:80 --namespace workshop-samples
+kubectl port-forward service/repo 9092:80 --namespace workshop-samples
```

Comment:
looks like this accidentally got included here - can we revert this?

Reply:
Actually, it's a change suggested by Claudio in order to avoid possible conflicts in case the user has standalone Pravega running on the default port 9090. https://github.com/pravega/workshop-samples/pull/4#discussion_r310777684

Reply:
I see - actually, i confused this with the code hub changes you made - sorry...  

In that case, this should be fine, although it is an unrelated change, it's small enough we can sneak it in this PR